src/Player.js

```diff
@@ -1,16 +1,16 @@
 class Player {
-  constructor(name, score = 0) {
+  constructor(name, score = 100) {
     this.name = name;
     this.score = score;
   }
 
-  incrementScore(points) {
-    return this.score += points;
-  }
+  // incrementScore(points) {
```

Comment:
No commented code going into master! We have version control for this. If you want the old code back, you can go through your commit history.